petab/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .core import *  # noqa: F403, F401
 from .lint import *  # noqa: F403, F401
+from .sbml import *  # noqa: F403, F401
```

Comment:
at some point might be good to explicitly declare exportable parts, to avoid e.g. doublings of namings. but i don't see that problem yet.

Reply:
Yeah currently I don't see a point in not exporting everything. The API is still in flow and the future will show what is commonly used and what not.